packages/api-specification/api/windows.ts

```diff
@@ -0,0 +1,313 @@
+import {Bounds, UnsubscribeFunction} from "./common";
+import {Base64ImageData, CaptureOptions} from "./system";
+import EventEmitter = NodeJS.EventEmitter;
+
+export interface WindowsAPI {
+
+    /** Returns the current window */
+    current: Window;

```

Comment:
If we have `getById()`, then we might want to have these as `getCurrent()` and `getFocusedWindow()` etc. this would match both Electron + OpenFinas well.